scripts/compare_benchmarks.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+# https://github.com/kubernetes/kube-state-metrics/blob/main/tests/compare_benchmarks.sh (originally written by mxinden)
+
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+[[ "$#" -eq 1 ]] || echo "One argument required, $# provided."
+
+REF_CURRENT="$(git rev-parse --abbrev-ref HEAD)"
+BASE_TO_COMPARE=$1
+
+RESULT_CURRENT="$(mktemp)-${REF_CURRENT}"
+RESULT_TO_COMPARE="$(mktemp)-${BASE_TO_COMPARE}"
+
+BENCH_COUNT=${BENCH_COUNT:-10}
+BENCHSTAT_CONFIDENCE_LEVEL=${BENCHSTAT_CONFIDENCE_LEVEL:-0.9}
+BENCHSTAT_FORMAT=${BENCHSTAT_FORMAT:-"text"}
+BENCH_PARAMETERS=${BENCH_PARAMETERS:-"-count 2000000 -batch-size 10000"}
+
+if [[ "${BENCHSTAT_FORMAT}" == "csv" ]] && [[ -z "${BENCHSTAT_OUTPUT_FILE}" ]]; then
+  echo "BENCHSTAT_FORMAT is set to csv, but BENCHSTAT_OUTPUT_FILE is not set."
+  exit 1
+fi
+
+function bench() {
+  local output_file
+  output_file="$1"
+  make build
```

Comment:
Pls refer to https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html
```suggestion
  $(MAKE) build
```

Reply:
This is not inside a make target, but I'll try with `$MAKE`

Reply:
that certainly does not work for me in bash
> $ $(MAKE) build
> bash: MAKE: command not found...